Tasks/google-play-release/GooglePlay.ts

```diff
@@ -399,7 +399,7 @@ async function updateTrack(
  * Uploads change log files if specified for all the apk version codes in the update
  * @param changelogFile
  * @param apkVersionCodes
- * @returns {*}
+ * @returns nothing
```

Comment:
For a different programming language (which I know better) I would write "returns void". I'm not sure if that's a correct expression for TypeScript.